### PR TITLE
Sensor list updates

### PR DESCRIPTION
**Pull request by fjoubert**
@theunsa please review and merge

added resize convenience buttons to graph in sensor list
fixed sensor list ordering, added config options for strategy types

note: next pull request massively refactors config display

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -96,6 +96,16 @@
...
         });
 
+        $rootScope.sensorListStrategyType = $localStorage['sensorListStrategyType'];
+        $rootScope.sensorListStrategyInterval = $localStorage['sensorListStrategyInterval'];
+
+        if (!$rootScope.sensorListStrategyType) {
+            $rootScope.sensorListStrategyType = 'event-rate';
+        }
+        if (!$rootScope.sensorListStrategyInterval) {
+            $rootScope.sensorListStrategyType = 3;
```
Comment (theunsa) [inline app/app.js]: I guess Interval is meant here ...

Comment (fjoubert) [inline app/app.js]: Good catch, yes

File: app/services/sensors-service.js [hunk 2]
```diff
@@ -159,7 +159,16 @@
...
-            api.sendSensorsCommand('set_sensor_strategy', [guid, resource_name, '', 'event-rate', 3, 3]);
+            api.sendSensorsCommand(
+                'set_sensor_strategy',
+                [
+                    guid,
+                    resource_name,
+                    '',
+                    $rootScope.sensorListStrategyType,
+                    $rootScope.sensorListStrategyInterval,
+                    $rootScope.sensorListStrategyInterval
```
Comment (theunsa) [inline app/services/sensors-service.js]: Why is Interval sent twice?

Comment (fjoubert) [inline app/services/sensors-service.js]: Event-rate strategy type requires a minimum and a maximum interval, which for now, we set as the same value. The second interval is ignored by the webserver for other strategy types where it is not applicable, like 'period' and in the case of 'event' both parameters are ignored.

Comment (theunsa) [inline app/services/sensors-service.js]: Got it.

**Pull request merged**